### DISCUSSION TITLE
How to Interact or click element inside frame/iframe

### DISCUSSION THREAD

**Discussion by ManimaranChadrababu** [Ginger-Automation/Ginger#2075] [Q&A] [answered]
Is there any way to click an element in side frame, even in recording steps that performed inside frame are not captured.
Any help is appreciated.

-Thanks inadvance

**Comment by JinendraGhodke** [comment, marked as answer]
In the browser action, we have switch frame operation. By switching to specific frame you can work on elements inside this frame.

If you will use Page object model feature, frame switching is automatically handled by Ginger
